release/build_release.sh

```diff
@@ -121,7 +121,11 @@ function main {
 
 		lc_time_run generate_api_source_jar
 
-		generate_poms
+		lc_time_run download_bnd_files 
+
+		lc_time_run generate_distro_jar
```

Comment:
@pgydev, shouldn't these 2 methods be invoked inside the method `generate_pom_release_dxp_distro` since they are requirements for the creation of the distro bom?

Reply:
@natocesarrego the `generate_distro_jar` method is not required for the creation of the distro bom. That being said the `download_bnd_files` is required by the `generate_distro_jar` so i should put it in there.